src/deserialize_sequence.cc

```diff
@@ -420,7 +384,7 @@ Sequence deserialize_sequence(const std::filesystem::path& path)
 
     std::vector<std::filesystem::path> steps;
     for (auto const& entry : std::filesystem::directory_iterator{path})
-        if (entry.is_regular_file() and entry.path().extension() == ".lua")
+        if (entry.is_regular_file() and entry.path().filename() != SEQUENCE_LUA_FILENAME)
```

Comment:
This would find all kinds of `.lua` files in the directory, but we are really only interested in those starting with `step_`, aren't we?

Reply:
Hmm, yes that is another interpretation of the for loop. I will change it to your suggestion because it is more explicit.